android/app/src/main/java/com/ledger/live/BackgroundRunnerPackager.kt

```diff
@@ -0,0 +1,19 @@
+package com.ledger.live;
+
+import com.facebook.react.ReactPackage
+import com.facebook.react.bridge.NativeModule
+import com.facebook.react.bridge.ReactApplicationContext
+import com.facebook.react.uimanager.ViewManager
+import java.util.*
+
+class BackgroundRunnerPackager : ReactPackage {
+    override fun createViewManagers(reactContext: ReactApplicationContext): List<ViewManager<*, *>> {
+        return emptyList()
+    }
+
+    override fun createNativeModules(reactContext: ReactApplicationContext): List<NativeModule> {
+        val modules: MutableList<NativeModule> = ArrayList()
+        modules.add(BackgroundRunner(reactContext))
+        return modules
+    }
+}
```

Comment:
Do we not run prettier on kt? I guess not

Reply:
I don't think so